syntax = "proto3";

package proto;

message User {
  int64 id = 1;
  bool had_trial = 2;
}

message UserId {
  int64 id = 1;
}

service UserService {
  rpc RegisterUser(UserId) returns (User) {}
  rpc GetUserByID(UserId) returns (User) {}
}

